Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.redhat.io/ubi8/ubi-minimal
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
Is it required to use `registry.access.redhat.com`?

Reply:
In order for the GH Workflow to pull the base image to build the app image and scan it. We need to use `registry.access.redhat.com` as it is the open image repo for Red Hat. `registry.redhat.io` requires us to provide credentials to pull the image to build and scan. It is the same image, just a different path to pull it.